go/libraries/doltcore/sqle/dfunctions/init.go

```diff
@@ -0,0 +1,11 @@
+package dfunctions
+
+import (
+	"github.com/src-d/go-mysql-server/sql"
+	"github.com/src-d/go-mysql-server/sql/expression/function"
+)
+
+func init() {
+	function.Defaults = append(function.Defaults, sql.Function1{Name: HashOfFuncName, Fn: NewHashOf})
```

Comment:
Isn't there a better way to install functions than adding them to the package var like this? You should use the session and catalog.MustRegister as in engine.go. Add a TODO if this is difficult on this pass

Reply:
added.